### ISSUE TITLE
Flexible content with preview mode not working

### ISSUE THREAD

**Issue by janzoutendijk** [wp-graphql/wp-graphql-acf#190]
Hi,

I'm trying to implement a preview version of my app, but I'm unable to get the updated flexible content from the graphql api. A wysiwyg field does work and the title also updates properly.

I'm using:
- ACF Pro 5.9.3
- WP GraphQL 1.0.1
- WPGraphQL for Advanced Custom Fields 0.4.0
- Wordpress 5.6

I'm using the GraphiQL interface to do the following query:

```graphql{
  post(id: 528, idType: DATABASE_ID, asPreview:true) {
    title
    blog {
      summary
      buildingblocks {
        __typename
        ... on Post_Blog_Buildingblocks_Text {
          text
        }
        ... on Post_Blog_Buildingblocks_Quote {
          quote
          cited
        }
      }
    }
  }
}
```

title and blog->summary are updating properly, but the content within the buildingblocks object is not.

Hopefully you can help! 👍

**Comment by jasonbahl** [comment]
@blauwhelm I believe this was resolved with #209. 

If not, let me know and we can re-open.

**Comment by janzoutendijk** [comment]
@jasonbahl Yes, it's resolved. Thank you very much!
